Source/Scene/Model.js

```diff
@@ -1936,6 +1938,17 @@ define([
             drawFS = 'uniform vec4 czm_pickColor;\n' + drawFS;
         }
 
+        if (model._upgrade10To20) {
```

Comment:
The version detection changed a bit with recent Model update. The source version should now be stored in `gltf.extras.sourceVersion`. Resolving this should fix the glTF 1 screenshots in https://github.com/AnalyticalGraphicsInc/cesium/pull/7017#issuecomment-427966855.